.editorconfig

```diff
@@ -0,0 +1,25 @@
+# This file defines a common editor configuration for a consistent code base.
```

Comment:
The go code formatting is done with `go fmt` so adding a default formatting might just create a conflict.
For the js code there is prettier which I think does the same thing?
The remaining is not going to be very useful as most of the editor you can use already use sensible default values.